mono/utils/mono-merp.c

```diff
@@ -261,6 +262,8 @@ get_merp_exctype (MERPExcType exc)
 			return "0x04000000";
 		case MERP_EXC_HANG: 
 			return "0x02000000";
+		case MERP_EXC_MANAGED_EXCEPTION:
+			return "0x80000000";
```

Comment:
@kdubau @alexischr Where do these magic hex numbers come from?

Reply:
Kyle reminded me that this is just consumed as a string on their end. So this lgtm